example/generator/generator_run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (c) 2018, Linaro Limited
```

Comment:
This is basically a new file, so in my opinion you could change the copyright to `Copyright (c) 2020, Marvell`